ZIGSIMPlus/Models/CommandAndCommandDataMediator.swift

```diff
@@ -9,6 +9,33 @@
 import Foundation
 
 public class CommandAndCommandDataMediator {
+    public func isAvailable(commandDataLabel: String) -> Bool {
+        switch commandDataLabel {
+        case LabelConstants.acceleration, LabelConstants.gravity, LabelConstants.gyro, LabelConstants.quaternion:
+            return MotionMonitoringCommand.shared.isAvailable()
+        case LabelConstants.touch:
+            return TouchMonitoringCommand.shared.isAvailable()
+        case LabelConstants.battery:
+            return BatteryMonitoringCommand.shared.isAvailable()
+        case LabelConstants.compass:
+            return CompassMonitoringCommand.shared.isAvailable()
+        case LabelConstants.pressure:
+            return AltimeterMonitoringCommand.shared.isAvailable()
+        case LabelConstants.gps:
+            return GpsMonitoringCommand.shared.isAvailable()
+        case LabelConstants.beacon:
+            return BeaconMonitoringCommand.shared.isAvailable()
+        case LabelConstants.proximity:
+            return ProximityMonitoringCommand.shared.isAvailable()
+        case LabelConstants.micLevel:
+            return MicLevelMonitoringCommand.shared.isAvailable()
+        case LabelConstants.ndi:
+            return NdiMonitoringCommand.shared.isAvailable()
+        default:
+            fatalError("Unexpected Command Data Label")
+        }
+    }
```

Comment:
How about making `CommandAndCommandDataMediator` singleton, instead of making all commands singleton.
In this case, we had to make commands singleton because commands belong to `CommandDataOutputPresenter` but the mediator can be called from other presenters.
If we make the mediator singleton and store the commands to it, we don't have to think about commands' life cycle.